about.html

```diff
@@ -0,0 +1,112 @@
+                <p>This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.</p>
+                <p>So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together</p>
+            </div>
+        </div>
+        <div class="hero-contact">
+            <p>Please contact us per Email for any further questions about CC Global Summit 2015! <span><b>ccsummit2015@cckorea.org</b></span></p>
+        </div>
+    </section>
+   </section>
+    <section class="about-main">
+        <div class="summit-logo"><h1 >Creative Commons Global Summit 2015 Logo</h1></div>
+        <p class="hero-p">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.</p>
+        <p class="hero-d">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.<br/>
+            After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.<br/>
+            After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+        <div class="about-logo">
+            <img src="images/about_logo 1.png" alt="about logo"/>
+        </div>
+    </section>
+    <section class="about-pass-event">
+        <div class="pass-event-head">
+            <div><h2>See the past CC Global Summits</h2></div>
+        </div>
+        <h3>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</h3>
+        <div class="conf-imgs">
+            <div class="img-1">
+                <h1>2011</h1>
+                <p> CC Global Summits in Warsaw</p>
+            </div>
+            <div class="img-2">
+                <h1>2013</h1>
+                <p> CC Global Summits in Buenos Aires</p>
+            </div>
+        </div>
+    </section>
+    <section class="partners-section">
+        <div class="partners-heading">
+            <h2>Partner</h2>
+        </div>
+        <div class="partners-brand">
+            <img src="images/mozilla 1.svg" alt="partners-logo" />
+            <img src="images/googlekorea 1.svg" alt="partners-logo" />
+            <img src="images/naver 1.svg" alt="partners-logo" />
+            <img src="images/daumkakaologo 1.svg" alt="partners-logo" />
+            <img src="images/airbnb 1.svg" alt="partners-logo" />
+        </div>
+    </section>
+    <section class="footer">
+        <img src="images/cck-logo1 1.svg" alt="creative common logo" />
+        <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+    </section>
```

Comment:
- [ ] I would recommend that you use a `<footer>` tag instead of a `<section>` tag at the bottom of your page. As it is more semantically  correct
- [ ] Also kindly adjust the background color of your footer section in the about me page to match the original design

![image](https://github.com/ctokoli/Conference-Capstone-Project/assets/98453979/9bd46f6e-171f-43be-b3ca-2850ea399143)